ts-obs/cmd/chunkIntervalReset.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+    "context"
+    "errors"
+	"fmt"
+    "os"
+
+	"github.com/spf13/cobra"
+    "github.com/jackc/pgx/v4/pgxpool"
+)
+
+// chunkIntervalResetCmd represents the chunk-interval reset command
+var chunkIntervalResetCmd = &cobra.Command{
+	Use:   "reset <metric>",
+	Short: "Resets the chunk interval for a specific metric back to the default",
+	RunE:  chunkIntervalReset,
+}
+
+func init() {
+	chunkIntervalCmd.AddCommand(chunkIntervalResetCmd)
+}
+
+func chunkIntervalReset(cmd *cobra.Command, args []string) error {
+    var err error
+
+    if len(args) != 1 {
+        return errors.New("\"ts-obs metrics reset-metric-chunk-interval\" requires 1 arguments")
+    }
+
+    if os.Getenv("PGPASSWORD_POSTGRES")  == "" {
+        return errors.New("Password for postgres user must be set in environment variable PGPASSWORD_POSTGRES")
+    }
```

Comment:
Postgres environment variables are more or less known/codified maybe we should use the same (PGPASSWORD) (https://www.postgresql.org/docs/current/libpq-envars.html)

Reply:
I was following the naming convention that we had in our notes when the Helm install happens, since we have an admin and a postgres user. May need to discuss whether the admin user is necessary.